extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/PrometheusEmitterConfig.java

```diff
@@ -78,8 +83,10 @@ public PrometheusEmitterConfig(
     this.port = port;
     if (this.strategy == Strategy.pushgateway) {
       Preconditions.checkNotNull(pushGatewayAddress, "Invalid pushGateway address");
+      Preconditions.checkNotNull(pushRateSec, "Invalid pushRateSec");
```

Comment:
Can this property have a default value if it's not configured by user?

Reply:
I changed the default value of flushPeriod to 15s, because the default scrape_interval of prometheus is the same